src/components/builder/i18n.tsx

```diff
@@ -122,7 +122,10 @@ export function ComponentTranslations<S extends AnyComponentSchema>({
                 </span>
               </td>
               <td>
-                <div aria-describedby={`component-translation-property-${property}`}>
+                <div
+                  aria-describedby={`component-translation-property-${property}`}
+                  style={{wordBreak: 'break-word'}}
```

Comment:
please use proper CSS for this instead of inline styles

`<div className="offb-content offb-content--allow-break">` seems a proper BEM approach:

* block 'content' for "plain" text content, so you can use this in multiple places
* modifier `allow-break` to explicitly signify we expect longs words to break

However, keep in mind that `word-break: break-word` is deprecated (see https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) - instead you should probably use `overflow-wrap: break-word;`, see https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Reply:
I've named this `offb-table` and `offb-table__content--allow-break`, because this also needed some styling for the table element.

And, because `word-break: break-word` is deprecated, i've changed it to `overflow-wrap: break-word`